incubator/redis-cache/values.yaml

```diff
@@ -0,0 +1,31 @@
+# Default values for redis-cache.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+##Statefulset related configurations ss.yaml
+replicaCount: 3
+redis:
+  image: redis
+  tag: 3.0-alpine
+  resources:
+    limits:
+      memory: 128Mi
+    requests:
+      memory: 128Mi
+microSentinel:
+  image: dhilipkumars/redis-sentinel-k8s
+  tag: latest
+makeSlave:
+  image: dhilipkumars/mk-redis-slave
+  tag: latest
+
+# Service related configurations
+service:
+  name: rd-port
+  port: 6379
+  type: ClusterIP
```

Comment:
`service.type` is not used but listed here. Remove it as it has to be `None` for the stateful set. On the other hand, I'm asking myself why you are using a stateful set. Wouldn't a deployment do as well, since there is no persistence?